src/AMD3.0

```diff
@@ -0,0 +1 @@
+/cygdrive/c/Program Files (x86)/AMD APP SDK/3.0
```

Comment:
As a note, the AMD APP SDK has been discontinued. It might be useful to rename `AMD3.0` to something more generic.